### PR TITLE
Create release-notes-1.0.0.md

### PR DESCRIPTION
**Pull request by dreacot**

### CODE REVIEW

File: docs/release-notes/release-notes-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Cryptopower is a self-custodial multi-coin wallet application designed for both desktop and mobile platforms. It offers support for creating wallets not only for Decred but also for other cryptocurrencies such as Bitcoin and Litecoin. The app is built using [Gio](https://gioui.org/), a Golang library for the implementation of cross-platform user interfaces.
```
Comment (Sirmorrison) [inline docs/release-notes/release-notes-1.0.0.md:1]:
```suggestion
Cryptopower is a self-custodial multi-coin wallet application designed for both desktop and mobile platforms. It offers support for creating wallets, not only for Decred, but also for other cryptocurrencies such as Bitcoin and Litecoin. The app is built using [Gio](https://gioui.org/), a Golang library for the implementation of cross-platform user interfaces.
```

File: docs/release-notes/release-notes-1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## Available Features
+
+- Multi-Coin support - Native SPV wallets for Decred, Bitcoin and Litecoin.
+ - Coin Control - It allows users to select exact addresses and outputs to be used in a transaction.
```
Comment (Sirmorrison) [inline docs/release-notes/release-notes-1.0.0.md:6]:
```suggestion
- Coin Control - It allows users to select exact addresses and outputs to be used in a transaction.
```

File: docs/release-notes/release-notes-1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+- CoinShuffle++ - CoinShuffle++ (CSPP) is a mixing protocol used to create Decred CoinJoin transactions.
+- Staking -  This allows users to purchase tickets via DCR’s [PoS (Proof-Of-Stake) consensus](https://docs.decred.org/proof-of-stake/overview/) implementation.
```
Comment (Sirmorrison) [inline docs/release-notes/release-notes-1.0.0.md:8]:
```suggestion
- Staking - This allows users to purchase tickets via DCR’s [PoS (Proof-Of-Stake) consensus](https://docs.decred.org/proof-of-stake/overview/) implementation.
```

File: docs/release-notes/release-notes-1.0.0.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- Governance - This allows users to vote on Decred Proposals, Consensus changes and Treasury Spendings.
+- Set gap limit - Users can choose a custom gap limit for use with address discovery.
+- Instant Exchange - Instant exchanging between assets using (Flyp.me, Godex, Changenow).
+- Fee Rates API - This allows for custom fee selection when sending BTC transactions.
+- Transaction note - This allows for a note/description when sending transactions.
+
+## Verifying the Release
+
+In order to verify the release, you'll need to have `gpg` or `gpg2` installed on your system. Once you've obtained a copy (and hopefully verified that as well), you'll first need to import the keys that have signed this release if you haven't done so already:
```
Comment (Sirmorrison) [inline docs/release-notes/release-notes-1.0.0.md:17]: This sentence is confusing especially the last part. It looks disconnected.

File: docs/release-notes/release-notes-1.0.0.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+gBjYVCa6INCaEw57dqB3CxBQloIZ4m7PY0ofYiKVD+PWidwztbfO4ee4MeaXVo6z
+NcUVH+fzcCZGgDO1sTuDOzOAdx5uvPmpyzMXzDAe5VYgLoHrsjDhJWsB9YZZjKhx
+/PN+YS3KGsrD8HMgNhAdn3/QkLD2OKl3/sW5kmuFmlywdLkocl8FmZlGyRd/XxVD
+u374UjAHT4NkQzbNV9DPKFeWk8yQq/xVj7wSezndFb/+YuGyz5Skji7mKTuoCjWS
+pawty5DgDQwzHHdhqGlT7Dnb/R9QMEyPvK12HaQ=
+=dt9F
+-----END PGP PUBLIC KEY BLOCK-----
+```
+
+Simply copy that entire text, save it as key.txt, and open a terminal.
```
Comment (Sirmorrison) [inline docs/release-notes/release-notes-1.0.0.md:63]: Copy the entire text, save it as 'key.txt,' and then open a terminal.

File: docs/release-notes/release-notes-1.0.0.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+In the terminal, use the cd command to navigate to the directory path where you’ve saved key.txt. Then use the `gpg --import` command.
```
Comment (Sirmorrison) [inline docs/release-notes/release-notes-1.0.0.md:65]: In the opened terminal, navigate to the directory where you saved 'key.txt' using the 'cd' command, and then execute the 'gpg --import' command.

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

File: docs/release-notes/release-notes-1.0.0.md (hunk 4 above)
Comment (dreacot) [reply docs/release-notes/release-notes-1.0.0.md:17]: how so?

Comment (ukane-philemon) [reply docs/release-notes/release-notes-1.0.0.md:17]: What do you think?

```
To verify the release, you must have either gpg or gpg2 installed on your system and import the keys that have signed this release. After installing gpg or gpg2, you can proceed with importing the key below if you haven't already done so:
```

File: docs/release-notes/release-notes-1.0.0.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+pawty5DgDQwzHHdhqGlT7Dnb/R9QMEyPvK12HaQ=
+=dt9F
+-----END PGP PUBLIC KEY BLOCK-----
+```
+
+Simply copy that entire text, save it as `key.txt`, and open a terminal.
+
+In the terminal, use the cd command to navigate to the directory path where you’ve saved `key.txt`. Then use the `gpg --import` command.
+
+(Note that Windows uses \ for directory paths, but Linux/macOS use /)
```
Comment (ukane-philemon) [inline docs/release-notes/release-notes-1.0.0.md:67]: Can we use back ticks to make the slash stand out?

File: docs/release-notes/release-notes-1.0.0.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+```
+gpg: assuming signed data in 'cryptopower-v1.0.0-manifest.txt'
+gpg: Signature made Thu 03 Aug 2023 17:08:06 WAT
+gpg:                using RSA key 5C26BFEC6C2466A528D5551CD05AC74F68976E52
+gpg:                issuer "release@cryptopower.dev"
+gpg: Good signature from "Cryptopower <release@cryptopower.dev>" [ultimate]
+```
+
+That will verify the signature of the manifest file, which ensures integrity and authenticity of the archive you've downloaded locally containing the binaries. Next, depending on your operating system, you should then re-compute the `sha256` hash of the archive with `sha256sum <filename>`, compare it with the corresponding one in the manifest file, and ensure they match exactly.
```
Comment (ukane-philemon) [inline docs/release-notes/release-notes-1.0.0.md:98]: > and ensure they match exactly

Can drop "exactly"? I think.

**Review by ukane-philemon — CHANGES_REQUESTED** [review]

File: docs/release-notes/release-notes-1.0.0.md (hunk 8 above)
Comment (dreacot) [reply docs/release-notes/release-notes-1.0.0.md:98]: i think "exactly" is an emphasis while still being correct English

File: docs/release-notes/release-notes-1.0.0.md (hunk 4 above)
Comment (dreacot) [reply docs/release-notes/release-notes-1.0.0.md:17]: hmmm, not a bad rephrase, failing to see the issue with the original paragraph

File: docs/release-notes/release-notes-1.0.0.md (hunk 8 above)
Comment (ukane-philemon) [reply docs/release-notes/release-notes-1.0.0.md:98]: np, we can keep it for "emphasis".

File: docs/release-notes/release-notes-1.0.0.md (hunk 4 above)
Comment (ukane-philemon) [reply docs/release-notes/release-notes-1.0.0.md:17]: > you'll first need to import the keys that have signed this release if you haven't done so already

I think @Sirmorrison was referring to this line. What keys? The key is just below the sentence but we can improve there.

Comment (dreacot) [reply docs/release-notes/release-notes-1.0.0.md]: please take another look

**Review by ukane-philemon — APPROVED** [review]

File: docs/release-notes/release-notes-1.0.0.md (hunk 4 above)
Comment (ukane-philemon) [reply docs/release-notes/release-notes-1.0.0.md]: All good.

**Pull request merged**